pkg/controller/gkenetworkparamset/gnpcontroller_validations.go

```diff
@@ -206,7 +206,7 @@ func (val *gnpNetworkCrossValidation) toCondition() v1.Condition {
 }
 
 // crossValidateNetworkAndGnp validates a given network and GNP object are compatible
```

Comment:
what is the definition of compatible?

Reply:
I remove the Network condition change out from here. For the original `crossValidateNetworkAndGnp` it means both GNP and corresponding Network should be valid based on the validation check below